packages/mermaid/src/docs/syntax/gitgraph.md

```diff
@@ -516,11 +516,11 @@ NOTE: Because we have overridden the `mainBranchOrder` to `2`, the `main` branch
 
 Here, we have changed the default main branch name to `MetroLine1`.
 
-## Orientation (v10.3.0+)
+## Orientation (v<MERMAID_RELEASE_VERSION>+)
```

Comment:
This should be (v10.3.0+) itself, please add the `v\<MERMAID_RELEASE_VERSION>+` tag to the BT header.